pretyping/coercion.mli

```diff
@@ -65,3 +57,22 @@ val inh_conv_coerce_rigid_to : ?loc:Loc.t -> program_mode:bool ->bool ->
     raises [Not_found] if no coercion found *)
 val inh_pattern_coerce_to :
   ?loc:Loc.t -> env -> cases_pattern -> inductive -> inductive -> cases_pattern
+
+
+type delayed_app_body
```

Comment:
Putting this in coercion is mostly so that reapply_coercions with idcoe isn't overly stupid